backend/routes/url.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const { getSurveyUrl } = require('../database');
+const router = express.Router();
+router.use(express.json());
+
+/**
+ * Route for requesting urls.
+ * @param {Object} req - The request object.
+ * @param {Object} req.body - The body of the request.
+ * @param {Object} res - The response object.
+ * @returns {Object} 200 - Success status.
+ * @returns {Object} 404 - Url not found.
+ */
+router.get('/api/url/:urlName', async (req, res) => {
```

Comment:
This could be renamed to something more descriptive, such as `/api/surveyUrl/:urlName`, but good for now.